src/controllers/update/update.manage.trusts.tell.us.about.the.former.bo.controller.ts

```diff
@@ -88,3 +96,11 @@ const getPageProperties = (trust: Trust, formData?: TrustHistoricalBeneficialOwn
   formData,
   errors
 });
+
+const getBackLink = (isIndividualOrLegalEntityInvolvedFlow: boolean) => {
```

Comment:
I think we could simplify some of the changes here.. 

Instead of, up at the top level of the controller, creating that flag and passing it through `getPageProperties` and `getBackLink`, we could reuse the trust that's been passed into `getPageProperties`, pass it through to `getBackLink`, which then reads from `trust.review_status?.reviewed_former_bos`?

Haven't looked yet but I expect similar implementation is in other controllers? Same suggestion for there..

Reply:
Changed